hail_search/web_app.py

```diff
@@ -8,6 +8,8 @@
 def _hl_json_default(o):
     if isinstance(o, hl.Struct) or isinstance(o, hl.utils.frozendict):
         return dict(o)
+    elif isinstance(o, set):
```

Comment:
Should this be `hl.set`?

Reply:
so it was an issue with a regular python set, sets are not actually json serializable and this solved the particular issue. But I can also add a case for a hail set

Reply:
actually, since we are only json serializing collected results we do not run into this case, as hail will have already converted all hail setexpressions to python sets. The only cases where we end up with hail-specific data types are those that hail provides which can exist as raw dat atypes and not expressions, which is just Interval, Struct, and frozendict - https://hail.is/docs/0.2/utils/index.html#utils

Reply:
yep, I think that's right.